scm/github/webhook.go

```diff
@@ -27,6 +28,14 @@ func (c *client) ProcessWebhook(request *http.Request) (*types.Webhook, error) {
 	h := new(library.Hook)
 	h.SetNumber(1)
 	h.SetSourceID(request.Header.Get("X-GitHub-Delivery"))
+
+	hookID, err := strconv.Atoi(request.Header.Get("X-GitHub-Hook-ID"))
+	if err != nil {
+		retErr := fmt.Errorf("unable to convert hook id to int64: %w", err)
+		return nil, retErr
```

Comment:
Thoughts on condensing this into one line?

```suggestion
		return nil, fmt.Errorf("unable to convert hook id to int64: %w", err)
```